science/bali-phy/Portfile

```diff
@@ -1,110 +1,39 @@
 PortSystem 1.0
-
+#PortGroup           github 1.0
 name                bali-phy
-version             2.1.1
-revision            5
-description         Bayesian Alignment and Phylogeny estimation
-long_description    BAli-Phy can estimate phylogenetic trees from sequence data when the alignment \
-                    is uncertain. Instead of conditioning on a single alignment estimate, BAli-Phy \
-                    accounts for alignment uncertainty by integrating over all alignments. \
-                    BAli-Phy does not rely on a guide tree because the alignment and the tree are \
-                    co-estimated. Therefore it can construct phylogeny estimates of widely \
-                    divergent sequences without bias toward a guide tree.
+version             3.0-beta6-20171223
+revision            1
+description         Estimation of phylogeny and multiple alignment via MCMC
+long_description    Estimation of phylogeny and multiple alignment via MCMC \
+                    BAli-Phy that estimates multiple sequence alignments and \
+                    evolutionary trees from DNA, amino acid, or codon sequences.\
+                    BAli-Phy uses MCMC and Bayesian  methods to estimate \
+                    evolutionary trees, positive selection, and branch lengths \
+                    while averaging over alternative alignments. \
+                     \
+                    BAli-Phy can also estimate phylogenies from a fixed \
+                    alignment (like MrBayes and BEAST) using substitution \
+                    models like GTR+gamma.  BAli-Phy automatically estimates \
+                    relative rates for each gene.
 categories          science
 platforms           darwin
-maintainers         nomaintainer
-homepage            http://www.biomath.ucla.edu/msuchard/bali-phy/
+license             GPL-2
+maintainers         gmail.com:benjamin.redelings openmaintainer
+homepage            http://www.bali-phy.org/
 
-master_sites        ${homepage}
-checksums           sha1 e72073a1c5b05c797668e476bfd8517594f074e6
+# By using a SHA, this should be completely reproduceable.
+fetch.type          git
+git.url             https://github.com/bredelings/BAli-Phy.git
+git.branch          217244e8dab7019d2501763ad0f97b8ef7473018
```

Comment:
Doesn't this work for you instead of fetching from git?
```
PortGroup           github 1.0

github.setup        bredelings BALi-Phy 217244e8
# just because of the strange case, otherwise the name should be set automatically; maybe all-lowercase works above as well
name                bali-phy
# usually this would be set automatically as well if using a tag, but '217244e8' is not a suitable version
version             3.0-beta6-20171223
```

Reply:
Not yet.  After I add some checksums, it still says "Error: tarball name is not as expected."

I suppose using the github portgroup has value, because it probably sets the livecheck url, and checks the repo name and other stuff, but using `fetch.type git` (a) just works and (b) is rather less verbose.  I used a long SHA so it seems like checksums might be redundant?